charts/azure-managed-cluster/values.yaml

```diff
@@ -47,14 +49,14 @@ controlplane:
   networkPlugin: "kubenet"
 
 #Only 1 Nodepool of mode System is allowed  
-agentpools:
+agentpools: 
  - name: agentpool0
    mode: System
-   sku: Standard_B4ms
-   nodecount: 2
+   sku: Standard_D4d_v5
+   nodecount: 5
    osDiskSizeGB: 100
  - name: agentpool1
    mode: User
    nodecount: 2
-   sku: Standard_B4ms
+   sku: Standard_D4d_v5
```

Comment:
Reason why this size?  I'd like to leave a smaller size for demo/cost purposes for default.  Above, we're putting in Standard_B2s.  These should all match.

Reply:
i'll update this to cover production economy

Reply:
I always use B's for my clusters, but anyway, this can be user-set